pkg/KubeArmorController/api/security.kubearmor.com/v1/common.go

```diff
@@ -96,7 +96,7 @@ type ProcessType struct {
 }
 
 type FilePathType struct {
-	Path MatchPathType `json:"path"`
+	Path MatchPathType `json:"path,omitempty"`
```

Comment:
I don't think this should be `omitempty`. Path is a required field provided matchPaths is used

Reply:
Ah, yes.